alpaca_data_cleaned.json

```diff
@@ -78112,7 +78112,7 @@
     {
         "instruction": "List three items in my grocery list.",
         "input": "Apples, eggs, and milk",
-        "output": "Apples, eggs, milk, carrots, onions, and tomatoes."
+        "output": "1. Apples\n2. Eggs\n3. Milk\n4. Carrots\n5. Onions\n6. Tomatoes"
```

Comment:
Weird instruction and output.

Reply:
what happened here was the output and input were swapped. Correct fix should be:
```
        "instruction": "List three items in my grocery list.",
        "input": "Apples, eggs, milk, carrots, onions, and tomatoes."
        "output": "1. Apples\n2. Eggs\n3. Milk"
```